docs/.templates/cli.md

```diff
@@ -63,7 +65,147 @@ If you have entered a command to the `build.beforeBuildCommand` property, this o
 
 {icon}
 
-[Tauri Icon Guide](../guides/features/icons.md)
+For more information, check out the complete [Tauri Icon Guide](../guides/features/icons.md).
+
+## `completions`
+
+<Command name="completions" />
+
+{completions}
+
+The Tauri CLI can generate shell completions for Bash, Zsh, PowerShell and Fish. To write the completions script to a file, run tauri completions --shell [bash|zsh|powershell|fish] > outputFile.
+
+Here are some instructions to configure Bash, Zsh and PowerShell. If you face an issue, please follow your shell's instructions instead. Note that it is recommended to check the generated completions script before executing it for security reasons.
+
+### Bash
+
+Get the Bash completions and move to a known folder:
+
+<Tabs groupId="package-manager">
+  <TabItem value="npm">
+
+```sh
```

Comment:
I think using `shell` instead of `sh` on these will make the leading `$` render, right?

Reply:
they even were `shell` when i copied the tab but i thought the word `shell` is kinda ugly so i replaced it. looks like you really can't trust my judgement anymore after a 3-4 day break 😂 

fixed it